docs/modding-basics/texture-replacement/index.md

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 2
+---
+
+# Texture Replacements
+
+Let's start with a simple texture replacement, understanding it will allow you to understand more complex asset replacements.
+
+## Prerequisites
+* [FModel](/docs/additional-resources/tools#fmodel)
+* [Repak](/docs/additional-resources/tools#repak-by-truman)
+* [DDSTools](/docs/additional-resources/tools#dds-tools) If you do not wish to install Unreal Engine.
+* [Unreal Engine 4.27](https://www.unrealengine.com/en-US/download) If you plan to do more than simple asset replacements.
+
+I recommend putting the tools in some directory like `Tools` before doing anything.
+
+## Replacing
+Find the texure you wish to replace, we'll replace the phone screen in this guide (`PAYDAY3/Content/Environment/_Common/Interactable/MobilePhone_01/Component/T_PhoneScreenWithAlpha_`). Go to the [Using FModel](/docs/basics/using-fmodel/) page to learn how.
+
+Edit the texture's png file as you'd like.
+
+There are a few different ways to replace assets, we'll focus on two for textures.
+
+### Using DDS-Tools
```

Comment:
I would place the Unreal Engine guide first, because it can be applied more generally, and DDS seems a bit more advanced.